CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## To Be Released
 
+### Changed
+- [Button, Card, Choice, ComboBox, Option, Placeholder, Rating, Select, Separator, TextInput, Toggle] Removed the 'Belle ' prefix from displayName
+
```

Comment:
No and yes :smile:

displayName is usually only used for debugging. Internally we also used it for propType checks. Some other people might have done the same. So if we want to comply to Semantic Versioning we would need to upgrade to 2.0 because it's a breaking change.

An alternative would be to fix this bug and go back to the old names: https://github.com/gaearon/babel-plugin-react-transform/issues/15

Might be the best solution. Since we didn't really change components we can merge this now and take care about it before we do another release.